API/Controllers/TournamentsController.cs

```diff
@@ -57,7 +57,8 @@ public async Task<IActionResult> CreateAsync([FromBody] TournamentSubmissionDTO
 
         if (await tournamentsService.ExistsAsync(tournamentSubmission.Name, tournamentSubmission.Ruleset))
         {
-            return BadRequest($"A tournament with name {tournamentSubmission.Name} for ruleset {tournamentSubmission.Ruleset} already exists");
+            return BadRequest(
+                $"A tournament with name {tournamentSubmission.Name} for ruleset {tournamentSubmission.Ruleset} already exists");
```

Comment:
I'm going to assume this was your formatter. I wonder why it behaves differently for everyone it's so odd... Maybe we need to dedicate an issue to taking another look at the editorconfig and setting some more rules

Reply:
Definitely the formatter, really not sure why it does this on my laptop.